editor-footnotes.php

```diff
@@ -52,3 +52,18 @@ function autoload( $cls ) {
 
 // Load the Editor class on admin pages
 add_action( 'admin_init', [ '\Editor_Footnotes\Editor', 'instance' ] );
+
+/**
+ * Allow span[data-footnote] to pass through post content.
+ *
+ * @param  array $kses Allowed HTML tags.
+ * @param  string $context Kses context.
+ * @return array
+ */
+function wp_kses_allowed_html( $kses, $context ) {
+	if ( 'post' === $context ) {
```

Comment:
Yeah I guess I really can't think of any legitimate reason why someone would *want* spans to be stripped out (and thus be upset when they started passing through again).